### ISSUE TITLE
Incompatibility with LucraftCore

### ISSUE THREAD

**Issue by Liquid-Donut** [LatvianModder/Project-EX#75]
ProjectEX tables and tablets crash when they render the LV Capacitor.

**Comment by Plasman888** [comment]
yup and they also cant work with matter override things with a battery percentage.

**Comment by darthvader45** [comment]
I have had this problem as well, with the matter overdrive battery.  Guess I'll have to use the ProjectE tablet from now on.
